test_runner/setup.cfg

```diff
@@ -0,0 +1,12 @@
+# Just trying to gather linter settings in one file.
```

Comment:
What's the role of all of these different tools? Do we need to run all of them, or are they alternatives to each other?

Reply:
That's a good question, actually. I use flake8 and yapf most of the time, but python-language-server also calls pyflakes, pycodestyle and god knows what else. I had to duplicate those settings to make these tools agree with each other.